docs/guidelines.md

```diff
@@ -0,0 +1,9 @@
+This issue intends to layout some guidelines on the transfer of [conda-recipes]( https://github.com/conda/conda-recipes ) and [anaconda-recipes]( https://github.com/ContinuumIO/anaconda-recipes ) to conda-forge. These aren't hard and fast rules. They are certainly up for discussion. However, it would be good to come up with a consensus going forward.
+
+Eventually all recipes from those repos should be proposed for addition here. It may be decided that a few don't actually belong or should not be supported anymore. For instance, a `vagrant` package seems a bit silly to me.
```

Comment:
Yeah, I see this as two sections almost. One is the proposal of how we transfer Continuum recipes and interact with the communities that have formed around them. The other is more technical details that could almost be a spec that motivates (or has motivated) the linter and reviews in general.